### PR TITLE
AutoML worker

### PR DESCRIPTION
**Pull request by dulacp**
Integrate changes from #120 with a few additional changes.

### CODE REVIEW

File: backends/core/workers.py [hunk 1]
```diff
@@ -949,3 +951,43 @@ def _execute(self):
...
+        'gcs_destination': {
+            'output_uri_prefix': self._params['destination_uri']
+        }
+    }
+    # Query model
+    response = prediction_client.batch_predict(
+        model_full_id, input_config, output_config)
+    print('Making batch prediction... ')
+    try:
+        result = response.result()
```
Comment (Slony) [inline backends/core/workers.py]: Would you mind using a 2-spaces indentation here?

File: backends/core/workers.py [hunk 2]
```diff
@@ -949,3 +951,43 @@ def _execute(self):
...
+    except (GoogleAPICallError, RetryError) as inst:
+        self.log_error('Error during batch prediction: %s' % inst)
```
Comment (Slony) [inline backends/core/workers.py]: Would you mind using a 2-spaces indentation here as well? :-)

**Comment by simonschellaert** [comment]
Hi! I think this pull request can be closed/deleted since the code above was merged (with some changes to allow for long-running jobs) in https://github.com/google/crmint/pull/173. Thanks!

– Simon

**Comment by Slony** [comment]
Closing in response to @simonschellaert comment.

@simonschellaert thank you for your contribution!

**Pull request closed without merging**